PlaygroundService/Grains/PlaygroundGrain.cs

```diff
@@ -1,14 +1,16 @@
 using System;
 using System.Diagnostics;
 using System.IO;
+using System.IO.Compression;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Orleans;
+using Orleans.Concurrency;
 
 namespace PlaygroundService.Grains;
-
+[Reentrant]
```

Comment:
Why do we need Reentrant for this grain?

Reply:
test code of DeactivateOnIdle
i have del it